### ISSUE TITLE
Import file not found 

### ISSUE THREAD

**Issue by k06a** [web3j/web3j-maven-plugin#17]
Have several solidity sources in `src/main/resources`:
```
StorageContract.sol
AccountContract.sol
```
Second one tries to import first one:
```
import './StorageContract.sol';
```
```
[ERROR] Failed to execute goal org.web3j:web3j-maven-plugin:0.1.4:generate-sources (default-cli) on project base-node: Could not compile solidity files
[ERROR] <stdin>:3:1: Error: Source "StorageContract.sol" not found: File not found.
[ERROR] import './StorageContract.sol';
[ERROR] ^-----------------------------^
[ERROR] -> [Help 1]
```

**Comment by k06a** [comment]
This also not works:
```
import 'src/main/resources/StorageContract.sol`
```

**Comment by k06a** [comment]
Even full path is not possible:
```
import '/Users/k06a/Projects/base-node/src/main/resources/StorageContract.sol';
```
```
[ERROR] Failed to execute goal org.web3j:web3j-maven-plugin:0.1.4:generate-sources (default-cli) on project base-node: Could not compile solidity files
[ERROR] <stdin>:3:1: Error: Source "/Users/k06a/Projects/base-node/src/main/resources/StorageContract.sol" not found: File outside of allowed directories.
[ERROR] import '/Users/k06a/Projects/base-node/src/main/resources/StorageContract.sol';
[ERROR] ^-----------------------------------------------------------------------------^
[ERROR] -> [Help 1]
```

**Comment by andrej-zirko** [comment]
Same issue here

**Comment by k06a** [comment]
@andrej-zirko right now I am writing all smart contracts in the same file, this works. But we need a good fix.

**Comment by h2mch** [comment]
there is raised already an issue in the solc library:
https://github.com/ethereum/solidity/issues/2928

**Comment by flefevre** [comment]
It is definietively a problem for me also.
I have dispatched my smart contract into several files to have a clear vision, but now it seems i need to fusion them again?

**Comment by ogurcan** [comment]
Same problem for me also...

**Comment by h2mch** [comment]
thx a lot to @MykolaMarkov. This issue is fixed in the latest version (0.3.0)

**Comment by Norphy** [comment]
@MykolaMarkov are we able to import from online sources? such as import "github.com/OpenZeppelin/openzeppelin-solidity/contracts/token/ERC721/ERC721Token.sol" because I get an error file not found.
